libs/designsystem/src/lib/components/card/card-header/card-header.component.scss

```
@@ -23,3 +33,32 @@ h4 {
   margin: 0;
   font-weight: font-weight('normal');
 }
+
+// Flagged card headers
+
+// key = level, value = color-name
+$notification-levels: (
+  'info': 'semi-light',
+);
+
+@each $color-name, $color-value in $notification-colors {
+  $notification-levels: map-merge(
+    (
+      $color-name: $color-name,
+    ),
+    $notification-levels
+  );
+}
+
+@each $level, $color-name in $notification-levels {
+  :host(.#{$level}) {
+    --kirby-card-header-background-color: #{get-color($color-name)};
+    --kirby-card-header-color: #{get-color($color-name + '-contrast')};
+    --kirby-card-header-padding: #{size('xxxxs')} #{size('xxs')};
+
+    h2 {
+      --kirby-card-header-line-height: #{line-height('n')};
+      --kirby-card-header-margin-bottom: 0;
+    }
+  }
+}
```

Consider moving shared declarations out of loop. This can be achieved by adding a util function to `_utils.css`:
```
@function keys-to-classes($map) {
  $classes: [];
  @each $key in map-keys($map) {
    $classes: append($classes, '.#{$key}', $separator: comma)
  }
  @return $classes;
}
```
```suggestion
@each $level, $color-name in $notification-levels {
  :host(.#{$level}) {
    --kirby-card-header-background-color: #{get-color($color-name)};
    --kirby-card-header-color: #{get-color($color-name + '-contrast')};
  }
}

$notification-levels-class-selectors: keys-to-classes($notification-levels);
:host(#{$notification-levels-class-selectors}) {
    --kirby-card-header-padding: #{size('xxxxs')} #{size('xxs')};
    
    h2 {
      --kirby-card-header-line-height: #{line-height('n')};
      --kirby-card-header-margin-bottom: 0;
    }
 }
```